common/autoware_test_utils/src/autoware_test_utils.cpp

```diff
@@ -122,12 +151,8 @@ LaneletMapBin make_map_bin_msg(
 
 LaneletMapBin makeMapBinMsg()
 {
-  const auto autoware_test_utils_dir =
-    ament_index_cpp::get_package_share_directory("autoware_test_utils");
-  const auto lanelet2_path = autoware_test_utils_dir + "/test_map/lanelet2_map.osm";
-  double center_line_resolution = 5.0;
-
-  return make_map_bin_msg(lanelet2_path, center_line_resolution);
+  return make_map_bin_msg(
```

Comment:
If you write 
```
get_absolute_path_to_lanelet_map("autoware_test_utils", "lanelet2_map.osm", center_line_resolution=5.0);
```
it will result in the same processing as the original code. However, since 5.0 is set as the default parameter, either way is fine for me.